src/lib/degradeableItems.ts

```diff
@@ -1,3 +1,4 @@
+import { Time } from '@sapphire/time-utilities';
```

Comment:
This should be
```ts
import { Time } from 'e';
```

We want to be sure we're always importing functionality from the same place to ensure compatibility and prevent unneeded regression issues. You can see **most** everywhere else in the code, we import `Time` from 'e'.

Reply:
fixed